.github/workflows/run-checks.yml

```diff
@@ -0,0 +1,88 @@
+name: "Run Checks"
```

Comment:
Nitpick [repeating]: for consistency, think we can omit the quotation marks from `name` fields.